### ISSUE TITLE
GZ my secret hideaway apartment 

### ISSUE THREAD

**Issue by blake1725** [DestinationHome/Destination-Home-Online#225]
Scene download error for EU account

**Comment by NagatoDEV** [comment]
Did you purchase it on your US account? Because it won't transfer over entitlements.

**Comment by xJeris** [comment]
I dont think this apartment works. I see a couple threads on it, like https://github.com/DestinationHome/Destination-Home-Online/issues/43

I can't even download it on PS3 (NA, OFW)

**Comment by xJeris** [comment]
This now downloads and lets me enter the scene.

**Comment by xJeris** [comment]
@blake1725 I think you opened this for someone else. Have they been able to confirm this is working or not? Thanks.

**Comment by blake1725** [comment]
this is working for EU now, yes i did open for someone
